codebundles/k8s-argocd-helm-health/runbook.robot

```diff
@@ -0,0 +1,98 @@
+*** Settings ***
+Documentation       This codebundle runs a series of tasks to identify potential helm release issues related to ArgoCD managed Helm objects.
+Metadata            Author    Nitin Madhok (nmadhok@icloud.com)
```

Comment:
For now the ideal Author text is a GitHub handle  - this will then render properly in RunWhen Local.